### PR TITLE
CorsFilterConfiguration logger should be static

### PR DESCRIPTION
**Pull request by clemgbld**
Fix https://github.com/jhipster/jhipster-lite/issues/7034

### CODE REVIEW

File: src/main/java/tech/jhipster/lite/security/infrastructure/primary/CorsFilterConfiguration.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 @Configuration
 class CorsFilterConfiguration {
 
-  private final Logger log = LoggerFactory.getLogger(CorsFilterConfiguration.class);
+  static private final Logger log = LoggerFactory.getLogger(CorsFilterConfiguration.class);
```
Comment (DamnClin) [inline src/main/java/tech/jhipster/lite/security/infrastructure/primary/CorsFilterConfiguration.java:15]: Java convention state that modifiers order should be : private static final (sonar would pick this up if I run the CI :))

**Comment by DamnClin** [comment]
Thanks! Can you sign the CLA and add Close #7034 in description so merging this PR will close de issue

**Comment by DamnClin** [comment]
You need to Add `Close` or `Fix` before the issue so it will be closed :)

**Comment by DamnClin** [comment]
Congratulation for your first contribution :).

**Comment by clemgbld** [comment]
> 

Thanks :)

**Pull request merged**